update-cdp.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+clean_devtools_directory() {
+    if [ -d "devtools-protocol" ] && { [ -f "devtools-protocol/browser_protocol.json" ] || [ -f "devtools-protocol/js_protocol.json" ]; }; then
+        rm -f devtools-protocol/*
+    fi
+}
+
+download_protocol_files() {
+    if ! wget -P devtools-protocol/ https://raw.githubusercontent.com/ChromeDevTools/devtools-protocol/master/json/browser_protocol.json https://raw.githubusercontent.com/ChromeDevTools/devtools-protocol/master/json/js_protocol.json; then
+        echo "Error: Failed to download files"
+        exit 1
+    fi
+}
+
```

Comment:
I think it's worth to take a commit hash from the ChromeDevTools repo and use it to fetch the protocol JSONs. We also need to update the latest supported version in the readme.

Reply:
if we want to make sure that this lib supports certain protocol version, we seems to need to add auto tests and pipelines. because in other case it would be a misleading to say that version is supported without proper testing. 
but if we can say its supported until it raises no errors, ok, we can include this protocol version. 
anyway i think synced autobump for protocol version will be more prefered.

Reply:
There are also two possible ways of implementing the protocol update:

1. through a bash script
2. by extending the functionality of cdpgen

I will do it through a bash script